android/modules/ui/src/js/listview.js

```diff
@@ -115,7 +115,13 @@ exports.bootstrap = function (Titanium) {
 					// widget does not exist, attempt to load namespace
 					widget = Module.main.require(namespace);
 				} catch (err) {
-					// do nothing...
+					// namespace does not exist, fall back to legacy behaviour
+					const lastDotIndex = namespace.lastIndexOf('.');
+					const proxy = eval(namespace.substring(0, lastDotIndex));
```

Comment:
<!--
  1 failure:  android/modules/u...
  1 warning:  android/modules/u...
  
  
  DangerID: danger-id-default;
  File: android/modules/ui/src/js/listview.js;
  Line: 120;
-->
- :no_entry_sign: android/modules/ui/src/js/listview.js line 120 – eval can be harmful. (no-eval)
- :warning: android/modules/ui/src/js/listview.js line 120 – eval with argument of type CallExpression (security/detect-eval-with-expression)